generator/src/test/java/com/scottlogic/deg/generator/cucumber/OfLength.feature

```diff
@@ -614,9 +614,10 @@ Feature: User can specify the length of generated string data using 'ofLength'
          { "field": "foo", "is": "ofLength", "value": 1 }
        ]}
        """
-    Then the following data should be included in what is generated:
+    Then the following data should be generated:
       | foo   |
       | null  |
+      | null  |
```

Comment:
This just seems wrong. Why would we write tests to assert that unnecessary, undesirable duplication occurs?

Reply:
Agreed, I have moved this and referenced back to issue #91